src/components/core/Input/Input.tsx

```diff
@@ -19,8 +20,19 @@ export interface Props
   containerProps?: StackProps;
 }
 
-const PandaInput = panda(ark.input, input);
-const PandaLabel = panda(ark.label);
+const { withProvider, withContext } = createStyleContext(input);
+
+// TODO extract primitives (this entire component will be refactored)
```

Comment:
This component looks a bit strange because previously, styles were not actually fully applied (now they are). Adjusting the styles of the input component is out of scope of this PR, but @hobbescodes and I talked earlier about refactoring it soon to clean it up, so I left this note here for now and we can tackle it soon